spec/services/conviction_decision_tree_spec.rb

```diff
@@ -53,6 +53,11 @@
     it { is_expected.to have_destination(:custodial_sentence, :edit) }
   end
 
+  context 'when the step is `conviction_type` equal discharge' do
```

Comment:
Not urgent, but we can consider refactor these specs as they grow, to read more like:

```ruby
context 'when the step is `conviction_type`' do
  let(:step_params) { { conviction_type: type } }

  context 'and type is `discharge`' do
    let(:type) { 'discharge' }
    it { is_expected.to have_destination(:discharge, :edit) }
  end

  context 'and type is `community_sentence`' do
    let(:type) { 'community_sentence' }
    it { is_expected.to have_destination(:community_order, :edit) }
  end
end
```

Reply:
👍 I was thinking that myself, i am going to refactor this once i have completed all the conviction types.